### PR TITLE
Add command to output locations

### PR DESCRIPTION
**Pull request by eleanorakh**
REF https://trello.com/c/QdTR0ajd/4288-raindar

### CODE REVIEW

File: lib/lita-raindar.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Raindar < Handler
       config :imgur_api_key
 
       route(/^weather ([a-zA-Z0-9\s]*)$/i, :radar, command: true, help: { "weather LOCATION" => "Returns GIF of recent weather radar" })
+      route(/^(radar locations)$/i, :radar_locations, command: true, help: { "radar locations" => "Returns list of available radar locations" })
```
Comment (yob) [inline lib/lita-raindar.rb:10]: The `(` and `)` are only required if you need to extract the value in the handler, so I _think_ it's safe to remove them from here

File: lib/lita-raindar.rb [hunk 2]
```diff
@@ -73,12 +74,71 @@ def radar(response)
...
+          Warruwi
+          Weipa
+          West Takone
+          Willis Island
+          Wollongong
+          Woomera
+          Wyndham
+          Yarrawonga
+        """
+      end
```
Comment (yob) [inline lib/lita-raindar.rb:91]: Could the `RADARS` Hash be used to build this programatically?

**Comment by yob** [comment]
Looks good! I have no further suggestions :ship:

**Comment by keithpitty** [comment]
:+1:

**Pull request merged**